Cargo.toml

```diff
@@ -11,18 +11,39 @@ build = "build.rs"
 include = ["src/**/*", "LICENSE", "README.md", "build.rs"]
 
 [features]
-default = ["server", "client", "tls", "noise", "websocket", "hot-reload"]
+default = [
+    "server",
+    "client",
+    "native-tls-support",
+    "noise",
+    "websocket",
+    "hot-reload",
+]
 
 # Run as a server
 server = []
 # Run as a client
 client = []
+
 # TLS support
-tls = ["tokio-native-tls"]
+tls-support = []
```

Comment:
what is this used for?

Reply:
This enables the common part of both "native-tls" and "rustls".

The use of this feature can be replaced by `#cfg[any(feature="native-tls", feature="rustls")]` in all places. Which is better?